presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -394,7 +396,7 @@ private static TestingPrestoServer createTestingPrestoServer(
                 extraModules,
                 baseDataDir);
 
-        String nodeRole = coordinator ? "coordinator" : resourceManager ? "resourceManager" : "worker";
+        String nodeRole = coordinator ? "coordinator" : resourceManager ? "resourceManager" : catalogServer ? "catalogServer" : "worker";
```

Comment:
Can we change this to `if else` to make it more readable?

Reply:
Sure done